docs/sources/alerting/unified-alerting/opt-in.md

```diff
@@ -10,6 +10,8 @@ Setting the `ngalert` feature toggle enables the new Grafana 8 alerting system.
 
 >**Note:** It is recommended to backup Grafana's database before enabling this feature. If you are using PostgreSQL as the backend data source, then the minimum required version is 9.5.
 
+>**Note:** This new system stores the notification log and silences on disk, hence it is a requirement to use persistent disks for using Grafana 8 alerts else the silences and notification log will be lost on restart and you might get unwanted or duplicate notifications.
```

Comment:
```suggestion
>**Note:** Since the new system stores the notification log and silences on disk, we require the use of persistent disks for using Grafana 8 alerts. Otherwise, the silences and notification log will get lost on a restart, and you might get unwanted or duplicate notifications.
```

Reply:
Can we avoid using 2 notes one after the other? If possible, move it to another section in the topic.

Reply:
There is only one another section, where this would not make sense. I can move it after the paragraph below, where it makes sense.

Reply:
Great! Thanks.